Source/SpaceparkAPI/Startup.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using SpaceparkAPI.Middleware;
+using SpaceParkAPI.Data;
+
+namespace SpaceparkAPI
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.AddMvc().AddXmlSerializerFormatters();
```

Comment:
Is AddMvc() actually needed?

its adding whole bunch of stuff that isn't really needed in the project, I would have possibly added on line 25
```suggestion
            services.AddMvc().AddXmlSerializerFormatters();
```

Reply:
the suggested change was meant to be services.AddControllers().AddXmlSerializerFormatters();
:'D

Reply:
Ahh, you are probably right!